src/js/review/SubmitController.jsx

```diff
@@ -76,15 +73,11 @@ class SubmitController extends React.Component {
     // like to know if they’re common
     const { isValid, errors } = isValidForm(form, pagesByChapter);
     if (!isValid) {
-      recordEvent({
-        event: `${trackingPrefix}-validation-failed`,
-      });
-      Raven.captureMessage('Validation issue not displayed', {
-        extra: {
-          errors,
-          prefix: trackingPrefix
-        }
-      });
+      const recordEvent = formConfig.recordEvent ?
+        formConfig.recordEvent.bind(formConfig) :
+        console.log.bind(console);   // eslint-disable-line no-console
+
+      recordEvent({ event: 'validation-failed-on-submit', errors });
```

Comment:
I suspected that duplication was intentional so I kept it. The event recording is as independent as possible to the app logic to avoid being affected by app bugs.